### PR TITLE
Feature/figma tokens ihor UuiDatePicker component

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: epam-promo/assets/theme.css [hunk 1]
```diff
@@ -160,7 +161,7 @@
     --input-bg-checked-hover-color: var(--primary-dark-color);
     --input-border-checked-disabled-color: var(--primary-light-color);
     --input-border-color: var(--gray40);
-    --input-bg-disabled-color: var(--gray5);
+    --input-bg-disabled-color: var(--input-bg-disabled-color);
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/assets/theme.css:164]: returned gray5

File: epam-promo/components/datePickers/RangeDatePickerBody.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import { withMods } from '@epam/uui-core';
 import { ReactComponent as ArrowIcon } from '../../icons/folding-arrow-18.svg';
 
 export function applyRangeDatePickerBodyMods() {
     return [
-        css.root,
+        // css.root,
         rangeCss.root,
-        calendarCss.root,
+        // calendarCss.root,
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/datePickers/RangeDatePickerBody.tsx:13]: in RangeDatePicker merge request it fixed

File: loveship/components/datePickers/RangeDatePickerBody.tsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 import { withMods } from '@epam/uui-core';
 import { ReactComponent as ArrowIcon } from './../icons/folding-arrow-24.svg';
 
 export function applyRangeDatePickerBodyMods() {
     return [
-        css.root,
+        // css.root,
         rangeCss.root,
-        calendarCss.root,
+        // calendarCss.root,
```
[the remark this answers is not in the document]
Comment (vik753) [reply loveship/components/datePickers/RangeDatePickerBody.tsx:13]: in RangeDatePicker merge request it fixed

File: loveship/components/datePickers/index.ts [hunk 4]
```diff
@@ -1,6 +1,3 @@
-export * from './Calendar';
-export * from './CalendarPresets';
 export * from './RangeDatePicker';
 export * from './DatePicker';
 export * from './RangeDatePickerBody';
-export * from './DatePickerBody';
```
[the remark this answers is not in the document]
Comment (vik753) [reply loveship/components/datePickers/index.ts:6]: Calendar, CalendarPresets and DatePickerBody now exports from uui

[the remark this answers is not in the document]
Comment (dsv-sergey) [reply loveship/components/datePickers/index.ts:6]: projects which use "import Calendar from '@epam/loveship'" will not understand

**Pull request closed without merging**